src/main/scala/com/exasol/cloudetl/kafka/KafkaConsumerProperties.scala

```diff
@@ -211,6 +212,39 @@ class KafkaConsumerProperties(private val properties: Map[String, String])
     props.toMap.asInstanceOf[Map[String, AnyRef]].asJava
   }
 
+  /**
+   * Returns a new [[KafkaConsumerProperties]] that merges the key-value pairs
+   * parsed from user provided Exasol named connection object.
+   */
+  final def mergeWithConnectionObject(exaMetadata: ExaMetadata): KafkaConsumerProperties = {
+    validateConnectionObject()
+    val connectionParsedMap =
+      parseConnectionInfo(SSL_KEYSTORE_LOCATION.userPropertyName, Option(exaMetadata))
```

Comment:
```suggestion
      parseConnectionInfo(SSL_KEYSTORE_PASSWORD.userPropertyName, Option(exaMetadata))
```

Reply:
Or use one of other password fields.

Reply:
According to the documentation for `parseConnectionInfo`, the username in the connection object is mapped to the property referenced in the first argument.. so it should probably not be a password field?

Reply:
I mean, I don't know if it's meaningful at all in the context of Kafka and certificate-based authentication, but the keystore location was the best I could come up with (eg. if you `CREATE OR REPLACE CONNECTION my_kafka TO '' USER 'buckets/.../.../mykeystore.p12' ...`, it would put the USER value as the SSL_KEYSTORE_LOCATION unless later overriden by the KV pairs in IDENTIFIED BY.

Reply:
Yes. Putting it does not hurt here since it is remapped from password field. Since we agreed above to keep the locations in the connection object you can keep as it is.